### ISSUE TITLE
ConEmu bug

### ISSUE THREAD

**Issue by Fanus** [tpope/vim-afterimage#6]
I'm getting this error when I try open an .png file using ConEmu on windows.
```
Error detected while processing function AfterimageReadPost:
line   34:
E484: Can't open file C:\Users\<username>\AppData\Local\BIV1342.tmp
```
The buffer ends up being empty :(

Is there perhaps a setting I am not aware of that could make this work?

**Comment by tpope** [comment]
Does this mean `'shell'` isn't `cmd.exe`? You're going to have to suggest a solution here.

**Comment by Fanus** [comment]
That is correct. I am rather new to vim so the best I can do is drop a few observations from my testing.
* Using `gvim.exe` I get the perfectly looking beautiful image.
* Using `cmd.exe` I get a ASCII looking image with the hex at the top of the buffer. 
* Using `ConEmu` the buffer stays empty and I get the error from my first comment.
* My tests were all when trying to open the same `.png` file

As a solution I can only think that there is something missing in the encoding of the characters in `cmd.exe` and `ConEmu`. Or perhaps the colors that are supported...
Are there any requirements in terms of the encoding and the colors when it comes to rendering the images in vim? 
I'm not even sure if `cmd.exe` is supported as images can obviously require a lot of colors and I know `cmd.exe `is quite limited in this regard. The main reason I use `ConEmu` is to get the correct rendering (amongst other things) for Solarized Dark. So I would assume it can handle this plugin's rendering.

**Comment by tpope** [comment]
All afterimage does is shell out to `convert`. My guess is shelling out is completely broken in your Vim, or at least any shelling out that uses `shellescape()`. Try `call system('echo '.shellescape(123))` and see if you get a similar error.
